draft-ietf-privacypass-architecture.md

```diff
@@ -174,14 +132,69 @@ The following terms are used throughout this document.
   attested to by the Attester.
 - Attester: An entity that attests to properties of Client for the
   purposes of token issuance.
+- Redemption context: The interactions and set of information shared
+between the Client and Origin.
+- Issuance context: The interactions and set of information shared
+between the Client, Attester, and Issuer.
```

Comment:
This is good! Should we also add the attestation context, or were you thinking that the issuance context would subsume the attestation context?

Reply:
I did view the issuance context as being a superset of the attestation context, but if you think it should be here too, that's fine!

Reply:
```suggestion
between the Client, Attester, and Issuer.
- Attestation context: The interactions and set of information shared between
the Client and Attester only, for the purposes of attesting the vailidity of the Client.
```